github/users.go

```diff
@@ -70,6 +70,14 @@ type User struct {
 	// repository. These are only populated when calling Repositories.ListCollaborators.
 	Permissions map[string]bool `json:"permissions,omitempty"`
 	RoleName    *string         `json:"role_name,omitempty"`
+
+	// Assignment identifies how a user was assigned to an organization role. Its
+	// possible values are: "direct", "indirect", "mixed". This is only populated when
+	// calling the ListUsersAssignedToOrgRole method.
+	Assignment *string `json:"assignment,omitempty"`
+	// InheritedFrom identifies the team that a user inherited their organization role
+	// from. This is only populated when calling the ListUsersAssignedToOrgRole method.
+	InheritedFrom *Team `json:"inherited_from,omitempty"`
```

Comment:
@stevehipwell the `inherited_from` field returns for me as an array which fails the unmarshalling, are you sure you got only a single team in the API result?

Reply:
I think the schema might have changed as it's now looks like an array, but I'm pretty sure I tested this.

Reply:
I want to change this field to an array can you verify it doesn't break anything for you? I want to know if we need to support both cases

Reply:
@stevehipwell I opened this: https://github.com/google/go-github/pull/3460